yaormfilter/boolfilter_test.go

```diff
@@ -0,0 +1,35 @@
+package yaormfilter_test
+
+import (
+	"github.com/geoffreybauduin/yaorm/yaormfilter"
```

Comment:
can you separate golang packages from community packages with a blank line?

Reply:
👍